modules/gdscript/gdscript.cpp

```diff
@@ -152,12 +152,21 @@ Variant GDScript::_new(const Variant **p_args, int p_argcount, Variant::CallErro
 	}
 
 	ERR_FAIL_COND_V(_baseptr->native.is_null(), Variant());
+    bool owner_valid = false; //owner can be invalidated in two ways: no base class or virtual base class
```

Comment:
Wrong indentation, see [Code style guidelines](http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html).

Reply:
I just installed the pre-commit hooks